### ISSUE TITLE
Dataset_list.html (single country page): filters not working

### ISSUE THREAD

**Issue by pzwsk** [GFDRR/open-risk-data-dashboard#453]
In example below, only risk category data should be selected

![image](https://user-images.githubusercontent.com/2378439/52719506-0e26a100-2fa6-11e9-9805-b81412cbfaf9.png)

**Comment by pzwsk** [comment]
Question to @CIMAManuel and @oncletom: is there an easy fix we can implement before Feb 28? If yes let's do it, otherwise I prefer to spend time on reviewing filtering feature.

Related to similar issue on Explore countries page.

Thanks

**Comment by oncletom** [comment]
Indeed.

There a 2 problems.

A quick one: the applicability filters are broken. The category ones are fine. It should be fairly easy to fix before the 28th.

Another problem is: even if the filter works, it happens `api/country` is not implemented as it used to be. The API parameters we used have no effect on the response. Eg: https://index.opendri.org/api/scoring/MW?category=Vulnerability

I checked for a quicker solution but we have no way to filter from the client-side. The quickest fix (both in implementation and UX) would be to provide the categories and applicability for each dataset of the response.

I think it's a quite sensible move to do, considering we do not have to recompute the score for each request (it was the reason why the filtering was done server-side).
